src/Elliptic/DiscontinuousGalerkin/NumericalFluxes/FirstOrderInternalPenalty.hpp

```diff
@@ -160,20 +202,9 @@ struct FirstOrderInternalPenalty<Dim, FluxesComputerTag,
  public:
   struct PenaltyParameter {
     using type = double;
-    // Currently this is used as the full prefactor `sigma` to the penalty term.
-    // This means it needs to be chosen large enough so that the scheme is
-    // stable everywhere. A good estimate is the the largest
-    // `sigma > N_points^2 / h` (see class documentation) over all elements in
-    // the domain. Choosing `sigma` the same everywhere means it is an
-    // overestimate on non-uniform meshes where elements are large or polynomial
-    // degrees are small, but this only affects the conditioning of the scheme,
-    // i.e. it will converge slower but to the same solution. When it becomes
-    // possible to communicate non-tensors (the element size and the number of
-    // collocation points on either side of the mortar), this option will be
-    // changed to be just the free parameter `C` multiplying `N_points^2 / h`.
-    // This will improve conditioning on non-uniform meshes.
```

Comment:
Why not add a comment saying what good values for C are now that N^2/h doesn’t have to be included?